components/search-bar/_search-bar.scss

```diff
@@ -0,0 +1,35 @@
+.active{
+  .sdc-input .sdc-input__input{
+    border-color: $blue;
+  }
+}
+
+.action-button{
+    cursor:pointer;
+    cursor: pointer;
+    position: relative;
+    float: right;
+    right: 11px;
+}
+
```

Comment:
Need to wrap all the styles below parent class: search-bar-container

In case the user type text that exceed the input width the icon will be covered with text.
Need to add specific padding-right: 30px; to: search-bar-input class.

The style: +.active{
 +  .sdc-input .sdc-input__input{
 +    border-color: $blue;
 +  }
 +}
is relevant for inputs, remove it from search scss